server/routes/api/browser.py

```diff
@@ -70,15 +63,15 @@ def get_property_labels(dcid):
 
 def get_sparql_query(place_id, stat_var_id, date):
     date_triple = "?svObservation observationDate ?obsDate ."
-    date_selector = "?obsDate"
+    date_selector = " ?obsDate"
     if date:
         date_triple = f'?svObservation observationDate "{date}" .'
         date_selector = ""
     sparql_query = f"""
-        SELECT ?dcid ?mmethod ?obsPeriod {date_selector}
-        WHERE {{ 
+        SELECT ?dcid ?mmethod ?obsPeriod{date_selector}
+        WHERE {{
```

Comment:
are these whitespaces significant (i.e. should we leave the ones after "." in the WHERE clause behind?)

Reply:
This is ok as the query is a multi-line string literal. Also verified locally.